packages/near-lake-primitives/src/types/core/index.ts

```diff
@@ -0,0 +1,8 @@
+export {
+    StreamerMessage,
+    BlockView,
+    BlockHeader,
```

Comment:
Since you've renamed `Block` -> `BlockView` I guess the same should be done with `BlockHeader` -> `BlockHeaderView`. However, it's not needed for `Shard`